core/scripts/go.mod

```diff
@@ -23,18 +25,19 @@ require (
 	github.com/shopspring/decimal v1.4.0
 	github.com/smartcontractkit/chainlink-automation v1.0.4
 	github.com/smartcontractkit/chainlink-common v0.2.3-0.20241001210038-dd59341432bd
+	github.com/smartcontractkit/chainlink/integration-tests v0.0.0-00010101000000-000000000000
```

Comment:
What causes this dependency? It seems backwards from what I would expect. Would another module for them both to share be ideal?